packages/kmod/kmod.spec

```
@@ -56,6 +82,7 @@ ln -s ../bin/kmod %{buildroot}%{_cross_sbindir}/modprobe
 %{_cross_bindir}/rmmod
 %{_cross_sbindir}/modprobe
 %{_cross_libdir}/*.so.*
+%{_cross_libexecdir}/kmod
```

Conventionally we "namespace" files under libexec by the owning package:

```suggestion
%dir %{_cross_libexecdir}/kmod
%{_cross_libexecdir}/kmod/kmod
```